enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java

```diff
@@ -923,6 +923,8 @@ public void updateUiState() {
 			splitRight.setRightComponent(logSplit);
 			logSplit.setLeftComponent(tabs);
 		}
+
+		splitRight.setDividerLocation(splitRight.getDividerLocation());
```

Comment:
I wonder why that fixes it... that's stupid haha

Reply:
I think that setting the component also sets the width to the minimum width of the component

Reply:
That would explain it resetting, yeah.